### PR TITLE
feature/1968-axa-input-text-add-onInput

### LINKED ISSUE #1968: axa-input-text add onKeyUp support for react
```
function myMethod(event) {
  console.log(event)
}

// ...

<axa-input-text onKeyUp={myMethod}>
```

Requested by @urswiss

### PR DESCRIPTION
**Pull request by urswiss**
Fixes #1968 

# Merge Checklist:
- [ ] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [ ] Tests are sufficient.
- [ ] Modifications still work in IE.
- [ ] Documentation is up to date.
- [ ] Changelog is up to date.
- [ ] Typescript typings for properties are up to date.
- [ ] Designers approved changes or no approval needed.

### CODE REVIEW

**Review by marek-polak — COMMENTED** [review]
I think we should rather use 'onInput' event instead - as mentioned in this question: https://stackoverflow.com/questions/574941/best-way-to-track-onchange-as-you-type-in-input-type-text, using `onkey*` events is not reliable, since you can right-click the field and choose Paste, and this will change the field without any keyboard input.

Instead, we should use `onInput` as answered few posts below (https://stackoverflow.com/a/26202266)
> It feels like onchange without the need to lose focus on the element. It is HTML5.
> It’s supported by everyone (even mobile), except IE8 and below. 

I noticed there is already support for `onInput` in mixin, so maybe all it needs to be done is to define it in typing for react?

**Comment by urswiss** [comment]
valid point. Ok if I adapt the PR accordingly, @raphaellueckl ?

File: src/components/10-atoms/input-text/index.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
       onChange: { type: Function, attribute: false },
       onFocus: { type: Function, attribute: false },
       onBlur: { type: Function, attribute: false },
+      onInputfieldKeyUp: { type: Function, attribute: false },
```
Comment (raphaellueckl) [inline src/components/10-atoms/input-text/index.js]: I would rename it to `onKeyUp`

**Review by danielnestler — DISMISSED** [review]
OK

**Comment by raphaellueckl** [comment]
Closing as there is another PR based on this one, merge conflics resolved.

https://github.com/axa-ch/patterns-library/pull/1975

**Pull request closed without merging**